snippets/floated-label.md

```diff
@@ -0,0 +1,82 @@
+### Floated label
+
+One of most common used pattern is input plus label. Using CSS to create floated label animation is achievable. And actually, it's simple with `:placeholder-shown` property.
+
+#### HTML
+
+```html
+<div class="floated-label-container">
+  <label for="input" class="input-label">
+    <input placeholder="&nbsp;" id="input">
+    <span class="label">Label</span>
+  </label>
+</div>
+```
+
+#### CSS
+
+```css
+.floated-label-container {
+  font-family: Arial, Helvetica, sans-serif;
+  height: 100px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.input-label input {
+  border: 0;
+  border-bottom: 2px solid #c8ccd4;
+  padding: 8px 0;
+  width: 250px;
+  background: #f5f6f9;
+}
+
+.input-label input:hover {
+  background: #f3f3f3;
+}
+
+.input-label input:focus {
+  background: none;
+  outline: none;
+}
+
+.input-label > .label {
```

Comment:
Add `pointer-events: none` so the cursor doesn't respond to the label

Reply:
Yeah, that's right. I didn't notice it. :) Thanks!